sky/clouds/aws.py

```diff
@@ -310,13 +310,18 @@ def make_deploy_resources_variables(
 
         image_id = self._get_image_id(r.image_id, region_name, r.instance_type)
 
+        disk_type = r.disk_type if r.disk_type is not None else 'low'
+
         return {
             'instance_type': r.instance_type,
             'custom_resources': custom_resources,
             'use_spot': r.use_spot,
             'region': region_name,
             'zones': ','.join(zone_names),
             'image_id': image_id,
+            'disk_iops': AWS.get_disk_iops(disk_type),
+            'disk_throughput': AWS.get_disk_throughput(disk_type),
+            'custom_disk_perf': AWS.enable_custom_disk_perf(disk_type),
```

Comment:
These functions seem only used inside this module, we can add the prefix `_`  to the function name to make them private. reference: https://google.github.io/styleguide/pyguide.html#3162-naming-conventions

Reply:
Actually, as mentioned below we can changes these separate calls to:

```suggestion
            **AWS._get_disk_specs(...)
```